<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Path_but">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选择视频文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="define_but">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>480</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pause">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>480</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_time">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>420</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clip_but">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>480</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>截取</string>
   </property>
  </widget>
  <widget class="QLabel" name="video_lab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>591</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="clip_lab">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>30</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="save_but">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>480</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择保存文件</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
